### PR TITLE
chore: make project existence check less strict

### PR DESCRIPTION
**Pull request by rgracey**
It's possible to configure a different (pretty) name via the "sonar-project.properties" file (sonar.projectName) and this would trip up the CI as it checks the project name (which is created differently in CI) and the project key.

Instead of checking both, just use the project key to check as it will be unique for the repository.

### CODE REVIEW

File: code-quality/action.yaml [hunk 1]
```diff
@@ -34,7 +34,7 @@ runs:
   steps:
     - uses: hipagesgroup/actions/github-env-vars-action@master
 
-    - uses: hipagesgroup/actions/setup-sonarcloud@master
+    - uses: hipagesgroup/actions/setup-sonarcloud@pl-740
```
Comment (koconder) [inline code-quality/action.yaml:37]: Revert

**Review by koconder — CHANGES_REQUESTED** [review]
Revert the version fix and then ill approve

File: code-quality/action.yaml (hunk 1 above)
Comment (rgracey) [reply code-quality/action.yaml:37]: Just testing for the moment over [here](https://github.com/hipagesgroup/job-assisted-posting-clefairy/actions/runs/4856136961)

**Comment by rgracey** [comment]
Working now 🚀 

https://github.com/hipagesgroup/job-assisted-posting-clefairy/actions/runs/4856136961/jobs/8655737034

**Review by koconder — APPROVED** [review]

**Pull request merged**